.github/workflows/check-docs-links.yaml

```diff
@@ -0,0 +1,19 @@
+on: [push, pull_request]
+name: Check docs links
+jobs:
+  check-docs-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
I think you need to add a step to build apidoc files first.

Quoting from the Action output:

```
docs/site/Executing-database-commands.md
ERROR	./apidocs/repository.defaultcrudrepository.execute.md
Stat docs/site/apidocs/repository.defaultcrudrepository.execute.md: no such file or directory
```

Here is the npm script to run:

```
npm run tsdocs
```

I am not sure what does it require:
- we may need to bootstrap all dependencies (run `npm install`)
- we may need to compile all packages (`npm run build`)

Perhaps @raymondfeng would know more?